### PR TITLE
Feature/dsl/user fixes

### PR DESCRIPTION
**Pull request by GiuliaB97**

### CODE REVIEW

File: src/test/scala/it/unibo/authsim/library/user/UserTest.scala [hunk 1]
```diff
@@ -36,7 +37,8 @@ class UserTest extends AnyWordSpec with should.Matchers{
   private var usernameSequence:Seq[String] = for(e<-userSequence) yield e.username
   private var passwordSequence:Seq[String]  = for(e<-userSequence) yield e.password
 
-  s"A user created with name '${name}' and password '{$password}'" should {
+  // TODO: these tests fail sometimes
```
Comment (asperan) [inline src/test/scala/it/unibo/authsim/library/user/UserTest.scala:40]: Comment to remove.

File: src/test/scala/it/unibo/authsim/library/user/UserTest.scala [hunk 2]
```diff
@@ -45,33 +47,34 @@ class UserTest extends AnyWordSpec with should.Matchers{
     }
   }
 
-  s"If a user created with a set of credential policies then user credentials" should  {
+  // TODO: also these tests fail sometimes
```
Comment (asperan) [inline src/test/scala/it/unibo/authsim/library/user/UserTest.scala:50]: Comment to remove.

**Review by asperan — CHANGES_REQUESTED** [review]

File: src/test/scala/it/unibo/authsim/library/user/UserTest.scala (hunk 1 above)
Comment (GiuliaB97) [reply src/test/scala/it/unibo/authsim/library/user/UserTest.scala:40]: Yes, sir.:D

File: src/test/scala/it/unibo/authsim/library/user/UserTest.scala (hunk 2 above)
Comment (GiuliaB97) [reply src/test/scala/it/unibo/authsim/library/user/UserTest.scala:50]: 👍🏻

**Review by asperan — APPROVED** [review]

**Pull request merged**